README.md

```diff
@@ -11,59 +11,41 @@
-### installation
-* Clone this repository
-* cd into the backend folder
-    ```bash
-    cd backend
-    ```
-* To install, compile and prepare the static resources run the following commands:
-    ```bash
-    npm run backend
-    npm run frontend
-    ```
-### Usage & Development
-#### Run the dev mode
-Dev mode allows you to run the framework in the browser, using vue cli for fast development cycles, and easy debug tools.
-To run it do the following:
-* In the backend folder run webpack or webpack-dev, then run the server.
-    ```bash
-    npm run webpack-dev
-    npm run ws:run
-    ```
-* In the frontend folder run serve
-    ```bash
-    npm run serve
-    ```
-* Open the broswer on localhost:8080 to access the framework.
-
-#### Run the VSCode extension
-* Start VSCode on your local machine, and click on open workspace. Select this repo folder.
-* On the debug panel choose "Run Extension", and click on the "Run" button.
-
-#### Enable usage analytics reporting from VS Code
-The tool collects non-personally identifiable information about your usage of the tool to improve its services. If you do not want the tool to collect your usage data, you can set the "Enable Sap Web Analytics" setting to "false". Go to File > Preferences > Settings (macOS: Code > Preferences > Settings) > Extensions > Application Wizard, and deselect the "Enable Sap Web Analytics" checkbox.
-
-#### Advanced scenarios
-To develop and contribute you can build and install each package separately. Instruction on each package in the dedicated readme.md file.
-* [Build & install the client](frontend/README.md)
-* [Build & install the backend](backend/README.md)
-* [Build & install the yeoman example generator](generator-foodq/README.md)
-
-## How to obtain support
+
+This npm [mono-repo][mono-repo] currently contains:
+
+- [VSCode Extension](./packages/backend) The backend part which communicates with Yeoman and the system. It runs as a VSCode extension or node.js application.
+- [Application Wizard UI](./packages/frontend) The Application Wizard as a standalone vue.js application.
+- [Yeoman generator sample](./packages/generator-foodq) Sample yeoman generator to show usages and test the platform.
+- [![npm-yeoman-ui-types][npm-yeoman-ui-types-image]][npm-yeoman-ui-types-url] [@sap-devx/yeoman-ui-types](./packages/types) Type signatures supposed to be used in the yeoman generators.
+
+[npm-yeoman-ui-types-image]: https://img.shields.io/npm/v/@sap-devx/yeoman-ui-types.svg
+[npm-yeoman-ui-types-url]: https://www.npmjs.com/package/@sap-devx/yeoman-ui-types
+
+## Support
+
 To get more help, support, and information please open a github [issue](https://github.com/SAP/yeoman-ui/issues).
 
+## Report an Issue
+
+To report an issue please use the github issue tracker. Please try to make sure you have these in your issue:
+
+- No duplicate
+- Reproducible
+- Good summary
+- Well-documented
+- Minimal example
+
+## Issue handling process
```

Comment:
I think there are templates you can add to .github directory to handle some of these docs related to issues.

Reply:
Great idea. We'll do it once I hope.